.github/ISSUE_TEMPLATE/docs_day.md

```diff
@@ -0,0 +1,58 @@
+---
+name: Consul docs day task
+about: This is a HashiCorp internal documentation task for the purpose of the Consul docs day event.
+labels: ['consul', 'placeholder']
```

Comment:
Maybe 
```suggestion
labels: ['type/docs', 'placeholder']
```

I'll send a Slack message to the community team about what to put instead of `placeholder`.